app/ante.go

```diff
@@ -0,0 +1,77 @@
+package app
+
+import (
+	"github.com/Sifchain/sifnode/x/clp"
+	"github.com/Sifchain/sifnode/x/faucet"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
```

Comment:
nit: I would add godocs to these ante-handler functions just so the reader knows the difference between each. They don't need to be too verbose.